app/views/support_interface/performance/index.html.erb

```diff
@@ -6,5 +6,6 @@
   <li><%= govuk_link_to 'Course options without vacancies', support_interface_course_options_path %></li>
   <li><%= govuk_link_to 'Applications with unavailable choices', support_interface_unavailable_choices_path %></li>
   <li><%= govuk_link_to 'Feature metrics', support_interface_feature_metrics_dashboard_path %></li>
+  <li><%= govuk_link_to 'Reasons for rejection', support_interface_reasons_for_rejection_dashboard_path %></li>
```

Comment:
I had envisioned this being in a separate tab like validations, but tbh it is a dashboard so it probably should go here. I think @stevehook's card should go into a separate tab though.

Reply:
Yeah we were a little, unsure on this one. We kind of placed it here for the time being. not against putting it into a separate tab though!